topics/evolution/tutorials/abc_intro_phylo/tutorial.md

```diff
@@ -0,0 +1,1287 @@
+
+The naive method for bootstrapping is called "non-parametric" and works by effectively resampling the patterns at each site in the alignment, creating a pseudo-alignment of the same total number of sites, then re-building the tree.
+
+IQTree has a very - *ultra* - fast bootstrapping method that is cleverer and works a bit better than the naive method. When we use IQTree in the next part of the tutorial, we will also do bootstrapping on the tree.  See {% cite 10.1093/molbev/msaa015 %} for details on the method.
+
+Bootstrapping can be done on any inference method. We will use the likelihood method that searches for the tree and branch lengths that maximises the likelihood for (1) our actual data, and then (2) for each of the pseudoreplicates, noting for each of these which branches occur in the best trees found.
+
+By keeping track of which branches occur in the best trees found for each of the pseudoreplicates, we can note how often the branches in the best tree for our *actual* data occur in the resampled data.  If they occur a lot -- say, 80% of the time or more -- then we can be fairly sure that that branch is well supported by the data.
+
+Bootstrap values therefore appear for each branch, and are most often expressed as a percentage or proportion. Branches at the leaves that occur in every possible tree so these would get 100% bootstrap values every time, and don't tell us anything.
+
+(Note: FastTree does not do bootstrapping natively, but can in conjuction with other tools ([see details](http://meta.microbesonline.org/fasttree/ )). It's fiddly to do this so we will not try it in this tutorial.)
+
+## Estimating a Maximum Likelihood tree with IQTree
+
+IQTree is a state-of-the-art cross-platform program that uses maximum likelihood to find optimal phylogenetic trees.
+It can perform model selection and bootstrapping.
+And it's on Galaxy!
+
+While this is running you might use your time to read the [Models of sequence evolution](#models-of-sequence-evolution) and [Bootstrapping](#bootstrapping) sections above.
+
+> <hands-on-title>Estimating a Maximum Likelihood tree with IQTree</hands-on-title>
+>
+> 1. Find the {% tool [IQTree](toolshed.g2.bx.psu.edu/repos/iuc/iqtree/iqtree/2.1.2+galaxy2) %} program in the tool finder.
+> 2. Load your aligned sequence data, i.e., the alignment from MAFFT.
+> 3. Leave the selection of data type as DNA.
+>    <!-- > 4. Ignore the Time Tree Reconstruction settings and Likelihood Mapping analysis settings. -->
+> 4. Under the **Modelling Parameters**, click **Automatic model selection**_ and then under **Use Custom Model** select "_(none))_" in the drop-down menu, and click the **Automatic model selection** menu title again to close it.
+> 5. Open the **Rate heterogeneity** menu and set the last option, **"Write maximum likelihood site ratios to .mlrate file"** to "Yes".
```

Comment:
as far as I can tell the `mlrate` file is not used again? is it necessary to add this output?